docs/REST API/rest-api.md

```diff
@@ -7404,21 +7404,60 @@ This allows you to easily configure scheduled queries that will impact a whole t
 ## Scripts
 
 - [Run script](#run-script)
-- [Get script result](#get-script-result)
-- [Run live script](#run-live-script)
 - [Add script](#add-script)
-- [Delete script](#delete-script)
+- [Get script result](#get-script-result)
+- [Upload script](#upload-script)
+- [Download script](#download-script)
 - [List scripts](#list-scripts)
-- [Get or download script](#get-or-download-script)
-- [Get script details by host](#get-script-details-by-host)
```

Comment:
why is this link being removed?

Reply:
Where is t supposed link? On the production current page it seems like it doesn't link to anything....

Reply:
@rachaelshaw I think we forgot to remove this link. We moved that to Hosts section. [See here](https://fleetdm.com/docs/rest-api/rest-api#get-hosts-scripts)